controls/charts/ReadMe.md

```diff
@@ -100,11 +96,11 @@ Following list of components are available in the package
    * [**Tooltip and Trackline**](https://ej2.syncfusion.com/demos/?utm_source=npm&utm_campaign=chart#/material/sparkline/default.html) - Supports interactive features such as tooltip and trackline.
 
 ### Smith chart
-   * [**Series**](https://ej2.syncfusion.com/demos/?utm_source=npm&utm_campaign=chart#/material/smithchart/default.html) - Supports any number of series to display data points.
```

Comment:
why did you remove the demo link reference?

Reply:
because FT link replaced here and demo taken into bottom part

Reply:
replaced with UG so, and now demo added